data/specials.inc

```diff
@@ -13,164 +13,164 @@ gSpecials:: @ 815FD60
 	def_special DoCableClubWarp
 	def_special ReturnFromLinkRoom
 	def_special CleanupLinkRoomState
-	def_special sub_8081770
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
+	def_special ExitLinkRoom
+	def_special NullFieldSpecial  @ Secret Base specials
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
 	def_special TryBattleLinkup
 	def_special TryTradeLinkup
 	def_special TryRecordMixLinkup
 	def_special CloseLink
 	def_special EnterColosseumPlayerSpot
 	def_special EnterTradeSeat
-	def_special Special_WiredCableClubTrade
-	def_special Special_CableClub_AskSaveTheGame
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
+	def_special StartWiredCableClubTrade
+	def_special CableClub_AskSaveTheGame
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
 	def_special SavePlayerParty
 	def_special LoadPlayerParty
 	def_special ChooseHalfPartyForBattle
 	def_special Script_ShowLinkTrainerCard
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
-	def_special nullsub_75
+	def_special NullFieldSpecial  @ Berry Tree specials
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
 	def_special IsEnigmaBerryValid
-	def_special ScrSpecial_GetTrainerBattleMode
-	def_special ScrSpecial_ShowTrainerIntroSpeech
-	def_special ScrSpecial_ShowTrainerNonBattlingSpeech
+	def_special GetTrainerBattleMode
+	def_special ShowTrainerIntroSpeech
+	def_special ShowTrainerCantBattleSpeech
 	def_special ScrSpecial_HasTrainerBeenFought
```

Comment:
maybe we can remove all `Special_` `ScrSpecial_` prefixes?

Reply:
I left a few in. Some can probably be removed, others like this one currently have duplicate names (`HasTrainerBeenFought` already exists)